index.html

```diff
@@ -67,9 +70,9 @@ <h2 class="project-title">Tonic</h2>
                    <li>css</li>
                    <li>javaScript</li>
                </ul>
-               <button class="action-button">See Project</button>
+               <button class="action-button see-project">See Project</button>
              </div>
-           </div>
+            </div>
 
          <div class="card">
              <img src="./img/Snapshoot Portfolio (1).png" alt="snapshot poortfolio" class="snapshot">
```

Comment:
- [ ] Having created your cards dynamically using `JavaScript`, I would kindly suggest you remove the code for the cards you have in your `HTML` file, With this you have show the best practice in javascript and as well follow the DRY method of coding practices.

Reply:
@brainconnect93 
Thanks for the time taken to review my code.  I do have some concerns. When you say I should remove the code for the card in my HTML file, do you mean I delete it? If yes, does that mean I have to create both the cards and the popup windows dynamically?